testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/observability/micrometer/multiple/BaseMultipleTestCase.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.wildfly.test.integration.observability.micrometer.multiple;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.client.WebTarget;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.testcontainers.api.DockerRequired;
+import org.jboss.arquillian.testcontainers.api.Testcontainer;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Do we include these annotations somewhere? I wasn't aware we did.

Reply:
The `jetbrains` anno? I think that's probably an IDE being overly helpful and I missed it.

Reply:
That was my assumption. It compiles so it must work, but just caught me by surprise :)